src/components/ErrorHandling/Sentry.tsx

```diff
@@ -46,6 +47,21 @@ const sendError = (
   withScope((scope) => {
     setScope(scope, { level: severity, context: metadata });
 
+    const { gqlErr, operationName } = metadata ?? {};
```

Comment:
Can we improve the type safety on these by adding `gqlErr` and `operationName` as option fields in the metadata struct on line 46.